src/dnf-plugins/subscription-manager.py

```diff
@@ -101,48 +110,50 @@ def config(self):
                         if os.path.basename(repo.repofile) != 'redhat.repo':
                             repo.disable()
                             disable_count += 1
-                    print(_('subscription-manager plugin disabled %d system repositories with respect of configuration in /etc/dnf/plugins/subscription-manager.conf') % (
-                                     disable_count))
+                    logger.info(
+                        _('subscription-manager plugin disabled %d system repositories with respect of configuration '
+                          'in /etc/dnf/plugins/subscription-manager.conf') % disable_count
+                    )
         else:
             logger.debug('Configuration file %s does not exist.' % default_config_file)
 
-    def _update(self, cache_only):
-        """ update entitlement certificates """
-        logger.info(_('Updating Subscription Management repositories.'))
-
-        # XXX: Importing inline as you must be root to read the config file
-        from subscription_manager.identity import ConsumerIdentity
+    @staticmethod
+    def _update(cache_only):
+        """
+        Update entitlement certificates and redhat.repo
+        :param cache_only: is True, when rhsm.full_refresh_on_yum is set to 0 in rhsm.conf
+        """
 
-        cert_file = str(ConsumerIdentity.certpath())
-        key_file = str(ConsumerIdentity.keypath())
+        logger.info(_('Updating Subscription Management repositories.'))
 
         identity = inj.require(inj.IDENTITY)
 
-        # In containers we have no identity, but we may have entitlements inherited
-        # from the host, which need to generate a redhat.repo.
-        if identity.is_valid():
-            try:
-                connection.UEPConnection(cert_file=cert_file, key_file=key_file)
-            # FIXME: catchall exception
-            except Exception:
-                # log
-                logger.info(_("Unable to connect to Subscription Management Service"))
-                return
-        else:
+        if not identity.is_valid():
             logger.info(_("Unable to read consumer identity"))
 
         if config.in_container():
             logger.info(_("Subscription Manager is operating in container mode."))
 
+        if cache_only is True:
+            log.debug('DNF subscription-manager operates in cache-only mode')
+
         if not cache_only and not config.in_container():
+            log.debug('Trying update entitlement certificates and redhat.repo')
```

Comment:
I think this message should be "Trying to update entitlement certificates and redhat.repo".

Reply:
Thanks. Updated.